assignment/impl/src/java/org/sakaiproject/assignment/impl/EmailUtil.java

```diff
@@ -65,8 +65,10 @@ public Map<String, Object> getReleaseGradeReplacements(Assignment a, String site
             Site site = siteService.getSite(siteId);
             map.put("siteTitle", site.getTitle());
             map.put("siteUrl", site.getUrl());
-            map.put("assignmentTitle", a.getTitle());
-            map.put("assignmentUrl", getAssignmentUrl(a));
+            map.put("title", a.getTitle());
+            map.put("url", getAssignmentUrl(a));
+            addIcon(map);
+            map.put("icon", getAssignmentUrl(a));
```

Comment:
Should this line be here? I don't get why you'd addIcon and then replace it with the Url?

Reply:
gone now. That was bullshit and you spotted it :)